build-files/install-packages.sh

```diff
@@ -27,34 +27,27 @@ fi
 header "Installing additional packages"
 # Update the package list
 msg "Updating package list..."
-sudo DEBIAN_FRONTEND=noninteractive apt-get -qq update -y
-
-# installing docker
-for pkg in docker.io docker-doc docker-compose docker-compose-v2 podman-docker containerd runc; do sudo apt-get remove $pkg; done
-
-# Add Docker's official GPG key:
-sudo DEBIAN_FRONTEND=noninteractive apt-get update -y 
-sudo DEBIAN_FRONTEND=noninteractive apt-get install ca-certificates curl -y
-sudo install -m 0755 -d /etc/apt/keyrings
-sudo curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
-sudo chmod a+r /etc/apt/keyrings/docker.asc
-
-# Add the repository to Apt sources:
-echo \
-  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu \
-  $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
-  sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
-sudo DEBIAN_FRONTEND=noninteractive apt-get update -y
-
-sudo DEBIAN_FRONTEND=noninteractive apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin -y
```

Comment:
are we completely getting rid of docker and switching to podman?

Reply:
Redhat is! RHEL has dropped support so I think the writing is on the wall. The podman-docker package gives you a docker command so that jobs that require it will still work. There are edge cases where docker != podman.